frontend/src/components/myPermissions/MyPermissionsTable.vue

```diff
@@ -0,0 +1,116 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import Column from 'primevue/column';
+import DataTable from 'primevue/datatable';
+import DataTableHeader from '@/components/managePermissions/table/DataTableHeader.vue';
+import { FilterMatchMode } from 'primevue/api';
+import {
+    TABLE_CURRENT_PAGE_REPORT_TEMPLATE,
+    TABLE_PAGINATOR_TEMPLATE,
+    TABLE_ROWS_PER_PAGE,
+} from '@/store/Constants';
+import { isLoading } from '@/store/LoadingState';
+
+import FamLoginUserState from '@/store/FamLoginUserState';
+
+const myPermissiosFilters = ref({
+    global: { value: '', matchMode: FilterMatchMode.CONTAINS },
+    'application': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'env': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'clientId': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'adminRole': {
```

Comment:
Hi Jota, let's not called it admin Role, because later on we'll use the same field for regular role as well, let's just call it role

Reply:
Hello, I'll change it to role.